cmd/helm/install.go

```diff
@@ -267,6 +267,10 @@ func (i *installCmd) run() error {
 		return prettyError(err)
 	}
 
+	if chartRequested.Metadata.Deprecated {
+		fmt.Fprintln(os.Stderr, "WARNING: This chart is deprecated")
```

Comment:
Why is this being sent to stderr. Warnings are typically sent to stdout. The only case I'm aware of is in deprecated code that's no longer used.

Reply:
https://github.com/helm/helm/pull/8273

Standard practice is to display warnings on stderr. That way programs that pipe the output of `helm template` into another (like `kubectl apply`) don't have mangled output. This is similar to other CLIs.

If someone were to redirect the output of your program to a file, would you want the warning placed in the file, or directed to the user?

Reply:
@mattfarina , thanks for fixing the problem of the failing tests and re-ran them. For the Stderr vs Stdout, are you ok with comment from @bacongobbler ? Are we good to resolve this conversation?

Reply:
@stantonxu we can resolve.

Reply:
@bacongobbler I went looking at v3 and there it's sent to stdout. https://github.com/helm/helm/blob/65ee1e0bc14907bb59daa4ac938a7ba145d5f2e5/cmd/helm/upgrade.go#L150-L152

We should be consistent. Does v3 need to change to stderr?

Reply:
Yes. Thanks for the reminder to go fix up https://github.com/helm/helm/pull/8277. That's the fix for Helm 3. :)

Reply:
@mattfarina Thanks.